spec/features/visibility_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe "View works via the UI" do
+
+  let(:user) { FactoryBot.create(:user) }
+  let(:admin_set) { FactoryBot.create(:admin_set) }
+  let(:public_work) { FactoryBot.create(:gw_work, admin_set: admin_set) }
+  let(:auth_only_work) { FactoryBot.create(:gw_only_work, admin_set: admin_set) }
+  let(:private_work) { FactoryBot.create(:gw_private_work, admin_set: admin_set) }
+  let(:solr) { Blacklight.default_index.connection }
+
+  before do
+    [public_work, auth_only_work].map { |work| solr.add(work.to_solr) }
```

Comment:
@dolsysmith 

Noticed that this doesn't seem to be adding `private_work` to Solr, so I think we might want to tweak these tests for handling private works.

If we want to test that one user can't view another user's private works, we could try instantiating two separate users, have one of those users be the creator of the `private_work`, log in the other user, and then verify that the `private_work` is not visible.

Reply:
Thanks for catching that! The private work now is saved to Solr, so I think the test should work as intended.

Reply:
Sweet, thanks for updating! 

For robustness sake, I think we'll want a test to verify that a user _can_ view their own private works, and _cannot_ view others' private works. I'm happy to take a stab at this today, or if you're interested - all you. 

I started writing some tests along these lines a while back on a branch [here](https://github.com/gwu-libraries/scholarspace-hyrax/blob/configure-user-permissions/spec/features/user_permissions_spec.rb), mostly just for demoing tests, which is why they never ended up in a PR.